document/core/valid/instructions.rst

```diff
@@ -459,6 +473,34 @@ The following auxiliary function denotes the number of lanes in a vector shape,
    }
 
 
+.. _valid-vternop:
+
+:math:`\shape\K{.}\vternop`
+...........................
+
+* The instruction is valid with type :math:`[\V128~\V128~\V128] \to [\V128]`.
+
+.. math::
+   \frac{
+   }{
+     C \vdashinstr \shape\K{.}\vternop : [\V128~\V128~\V128] \to [\V128]
+   }
+
+
+.. _valid-rlaneselect:
+
+:math:`\shape\K{.}\RLANESELECT`
+...............................
+
+* The instruction is valid with type :math:`[\V128~\V128~\V128] \to [\V128]`.
+
+.. math::
+   \frac{
+   }{
+     C \vdashinstr \shape\K{.}\vternop : [\V128~\V128~\V128] \to [\V128]
```

Comment:
vternop -> RELAXEDLANESELECT.

Though, can this not actually be modelled as a ternop?

Reply:
It can for validation, but it doesn't match the usual ternop semantics, because it's not a lane-wise operation.
So if we group RELAXEDLANESELECT into vternop, it matches the vternop semantics, which is wrong.